my-app/src/chatWindow/services/chatMapperService.js

```diff
@@ -0,0 +1,23 @@
+class ChatMapperService {
+	mapMyMessage = (textMyMessage, AUTHOR) => {
```

Comment:
if I understand correctly, it should be as mapToDTO (as we send dto model to the backend only)